Utilization of event data fields in SelEntry class

Arcticae opened the pull request.

Arcticae commented:
It's the same way i did it in the previous PR, and IPMI documentation defines these 3 variables as 3 separate fields, so i thought it would be more readable that way, although array would be fine with me too. If you think it's more convenient, let me know and i'll edit the PR.

hthiery commented:
yes ... sorry for the confusion. I think it would be better to have these kind a values as list/array like value

Arcticae commented:
@hthiery I've changed event_data to store it as an array of integers (more useful for further decoding than plain string), and also corrected the event_type decoding, as it was only taking 6 younger bits of the event (mask was 0x3f), instead of 7 (now mask is 0x7f). Would appreciate if you reviewed the changes and merged them :)
